src/com/hms_networks/americas/sc/realtimedata/RealTimeTagDataPointManager.java

```diff
@@ -0,0 +1,133 @@
+package com.hms_networks.americas.sc.realtimedata;
+
+import java.util.ArrayList;
+
+import com.ewon.ewonitf.TagControl;
+import com.hms_networks.americas.sc.datapoint.DataPoint;
+import com.hms_networks.americas.sc.datapoint.DataPointBoolean;
+import com.hms_networks.americas.sc.datapoint.DataPointDword;
+import com.hms_networks.americas.sc.datapoint.DataPointFloat;
+import com.hms_networks.americas.sc.datapoint.DataPointString;
+import com.hms_networks.americas.sc.logging.Logger;
+import com.hms_networks.americas.sc.taginfo.TagInfo;
+import com.hms_networks.americas.sc.taginfo.TagType;
+
+/**
+ * This class will hold a list of data points for each tag. One instance of the class is made per
+ * tag.
+ *
+ * @author HMS Networks, MU Americas Solution Center

```

Comment:
@ since missing

Reply:
I can do a readme update in another PR to show a version update. version set to 2.0